src/tribler/core/upgrade/upgrade.py

```diff
@@ -243,10 +243,12 @@ def add_column(db, table_name, column_name, column_type):
 
             self._logger.info(f'{version.current}->{version.next}')
 
-            add_column(db=mds._db, table_name='ChannelNode', column_name='tag_processor_version', column_type='INT')
-            add_column(db=tags.instance, table_name='TorrentTagOp', column_name='auto_generated', column_type='BOOLEAN')
+            if tags is not None:
```

Comment:
Thank you for the suggestion! I think it is better to have this check repeated in future upgrades.

Right now, the tags database does not have a version, and we rely on the metadata store version. If we create the tags database during the upgrade, it will already have the latest schema, and the subsequent upgrades should have the logic to skip the upgrade. So, creating the database in the upgrade will not reduce the number of checks.